src/quadratic_model.jl

```diff
@@ -136,6 +136,9 @@ function complete_quadratic_model!(Q::QmIGE, c, box::Box{T,N}, f::Function, x0,
             elseif xcur >= u
                 u, xcur = xcur, (l + xcur)/2
             end
+            if l == u
```

Comment:
Would `l == u || abs(u-l) < epswidth((l, u))` be more robust?

Reply:
Ah yes, for sure!  But in that case I think we don't need `l == u` at all; ` abs(u-l) < epswidth((l, u))` should include the case when they're equal.  I'll amend the commit.